examples/developer-tools/src/index.html

```diff
@@ -45,21 +45,29 @@
                 </label>
                 or drag it here
               </p>
-              <p id="file-upload-warning" class="warning-message">
+              <p id="file-upload-warning" class="warning-message hidden">
                 File could not be parsed. Make sure you're uploading the file
                 you downloaded from the legacy Block Factory.
               </p>
             </div>
           </div>
           <div slot="actions">
-            <md-text-button
-              id="file-upload-close"
-              form="form-id"
-              value="cancel">
+            <md-text-button id="file-upload-close" value="cancel">
               Cancel
             </md-text-button>
           </div>
         </md-dialog>
+        <md-dialog id="upload-results-modal">
```

Comment:
What is this I don't even.

Reply:
Can't tell what part of this you are reacting to, is it the custom element? They're fun and also funfact you can't put blockly inside one :)